src/app_service/service/community/service.nim

```diff
@@ -622,11 +622,11 @@ QtObject:
   proc getAllChats*(self: Service, communityId: string, order = SortOrder.Ascending): seq[ChatDto] =
     ## Returns all chats belonging to the community with passed `communityId`, sorted by position.
     ## Returned chats are sorted by position following set `order` parameter.
-    if(not self.joinedCommunities.contains(communityId)):
+    if(not self.allCommunities.contains(communityId)):
       error "trying to get all community chats for an unexisting community id"
       return
 
-    result = self.joinedCommunities[communityId].chats
+    result = self.allCommunities[communityId].chats
```

Comment:
We need to make sure that this won't break anything.
I've seen places in the past where `joinedCommunities` and `allCommunities` would not kept 100% in sync. 
Just raising this here to be double checked there isn't a case where one `chats` looks different than the other.

Reply:
Good point. I will revert it to not risk the regression. Created https://github.com/status-im/status-desktop/issues/9570